### ISSUE TITLE
Telephone-like numbers need formatting that they aren't telephone numbers

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4228]
When viewing a page on iOS in Safari (and other smartphone devices?) any number which could be a telephone number gets formatted as a link to 'call' that number.  This is an a11y issue, as it makes the content potentially confusing.  In particular issn numbers are being confused this way, but likely there are more kinds of numbers which could be affected.

Identified in Screen reader testing (#4194):
- Sample 3, item 55. the issn number is picked up by ios Safari as if it were a telephone number (turns it into a link and offers to call it). This behaviour can, and should be overridden in the HTML. And similar overrides considered for all numbers we use that won't be telephone numbers!

It appears this situation can be avoided:
``` <meta name="format-detection" content="telephone=no"/> ```. see [Apple Documentation](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html#//apple_ref/doc/uid/TP40008193-SW5)

**Comment by ajrbyers** [comment]
Marked this as in review based on submitted PR: https://github.com/openlibhums/janeway/pull/4558
